File: algorithms/interview-preparation-kit-2/php/code-response/spec/CharCode/CharCode.spec.php
<?php

use CharCode\CharCode;

describe('Char Code', function () {
  it('adding char code of "a"', function () {
    expect(CharCode::sum('a'))->toEqual(97);
  });

  it('adding char code of "b"', function () {
    expect(CharCode::sum('b'))->toEqual(98);
  });

  it('adding char code of "A"', function () {
    expect(CharCode::sum('A'))->toEqual(65);
  });

  it('adding char code of "ifpb"', function () {
    expect(CharCode::sum('ifpb'))->toEqual(417);
  });

  it('adding char code of "lorem"', function () {
    expect(CharCode::sum('lorem'))->toEqual(543);
  });

  it('addinging Codechar code of "lorem ipsum"', function () {
    expect(CharCode::sum('lorem ipsum'))->toEqual(1133);
  });
});

File: algorithms/interview-preparation-kit-2/ruby/code-response/spec/CharCode/char_code.spec.rb
require_relative "../../src/CharCode/char_code"

describe CharCode do
	describe ".sum" do
		context 'adding char code of "a"' do
			it "return 97" do
					expect(CharCode.sum("a")).to eql(97)
			end
		end
		context 'adding char code of "b"' do
			it "return 98" do
					expect(CharCode.sum("b")).to eql(98)
			end
		end
		context 'adding char code of "A"' do
			it "return 65" do
					expect(CharCode.sum("A")).to eql(65)
			end
		end
		context 'adding char code of ifpb"' do
			it "return 417" do
					expect(CharCode.sum("ifpb")).to eql(417)
			end
		end
		context 'adding char code of "lorem"' do
			it "return 543" do
					expect(CharCode.sum("lorem")).to eql(543)
			end
		end
		context 'adding char code of "lorem ipsum"' do
			it "return 1133" do
					expect(CharCode.sum("lorem ipsum")).to eql(1133)
			end
		end
	end
end
